app/src/main/java/br/org/eldorado/millelds/ui/MainActivity.kt

```diff
@@ -32,7 +33,21 @@ class MainActivity : AppCompatActivity() {
     }
 
     private fun setupViews() {
-        binding.recyclerView.adapter = ProductListAdapter(productList)
+        binding.recyclerView.adapter = ProductListAdapter(productList).apply {
+            onItemClickListener = { product ->
+                Snackbar.make(
+                    binding.root,
+                    "Clicked product ${product.name}",
+                    Snackbar.LENGTH_SHORT
+                ).show()
```

Comment:
i don't think that you need to use this snackbar since you're changing the activtiy